runtime/src/main/java/org/corfudb/runtime/collections/streaming/StreamPollingScheduler.java

```diff
@@ -151,10 +154,10 @@ public void addTask(@Nonnull StreamListener streamListener, @Nonnull String name
                 throw new StreamingException(
                         "StreamingManager::subscribe: listener already registered " + streamListener);
             }
-            StreamingTask task = new StreamingTask(runtime, workers, namespace, streamTag, streamListener,
-                    tablesOfInterest, lastAddress, bufferSize);
+            StreamingTask task = new StreamingTask(runtime, workers, nsToStreamTags, streamListener,
+                    nsToTables, lastAddress, bufferSize);
             allTasks.put(streamListener, task);
-            log.info("addTask: added {} {} {} address {}", streamListener, namespace, streamTag, lastAddress);
+            log.info("addTask: added {} for {} address {}", streamListener, nsToStreamTags, lastAddress);
```

Comment:
Do we need to list the addTask is in StreamPollingScheduler to help in debugging scenarios?

Reply:
I don't think we need to list all the tasks since every streaming task is isolated.
We can discuss more about this offline.